README.md

```diff
@@ -47,187 +18,141 @@ After you're finished please remove all the comments and instructions!
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Javascript Capstone Project] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**Javascript Capstone Project** is a an application that fetches tv shows from TVMaze API and also has a feature of posting and fetching Likes, Comments and Reservations from an involvement API.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+    <li>Webpack</li>
+    <li>Jest</li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **Web pack deployed**
+- **JavaScript dynamically displays list**
+- **Fetches TV Shows details from TVMaze API**
+- **Posts Likes, Comments and Reservations in an Involvement API**
+- **Fetchs Likes, Comments and Reservations from an Involvement API**
+- **Comment and Reservations PopUp Deployed**
+- **Item, Like, Comment and Reservation Counter Deployed**
+- **All Counters Tested with Jest**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Video presentation -->
+## Video Presentation <a name="video-presentation"></a>
+[Click here](https://drive.google.com/file/d/1yALLexsypFizmaAbXW6Rr8J1zhxZVQnp/view?usp=sharing)
 
```

Comment:
- [ ] Clear and engaging presentation, kindly highlight some interesting piece of code or something you built that you are very proud of in accordance to the project requireent. 👍